src/main/java/com/google/sps/model/GmailClient.java

```diff
@@ -54,6 +54,16 @@
   Message getUserMessageWithMetadataHeaders(String messageId, List<String> metadataHeaders)
       throws IOException;
 
+  /**
+   * Gets the word count in the unread emails from the past days. The period length to go through
+   * the emails is specified in the parameter passed in the method
+   *
+   * @param numberDaysUnread the number of days to look through the emails
+   * @return an integer representing the word count in the unread emails
+   * @throws IOException if an issue occurs with the Gmail service
+   */
+  int getWordCount(int numberDaysUnread) throws IOException;
```

Comment:
I would abstract this into a different helper than this one - this is meant to be for GET requests from Gmail, while this deals with the processing of information. Consider creating a `PlanMailResponseHelper` class, similar to what was done for Gmail. Or other names if you have any ideas

Reply:
Drive-by comment - I agree about putting these methods in a separate utility. 

Also, it's not immediately obvious that the implementation of this method is tested. A separate utility with separate tests would make that obvious.

Reply:
I agree, I refactored my implementation to put this in `PlanMailResponseHelper`. As for testing the method itself, that requires creating Message objects to pass in, and I am having run-time errors when I do that. I am tabling it so that I can focus on the rest of the review for now.

Reply:
Sure 

RESOLVED